deepinv/tests/test_datasets.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from deepinv.datasets import DIV2K
+
+
+def test_download_div2k():
```

Comment:
we need to remove the dataset after performing the test, but i am not sure how to do it properly as intented by pytest (can we define a function that will be called after all the test for clean-up ?)